github_connector/models/abstract_github_model.py

```diff
@@ -0,0 +1,207 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016-Today: Odoo Community Association (OCA)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import base64
+import urllib
+import logging
+from datetime import datetime
+
+from openerp import tools, api, fields, models, exceptions, _
+from .github import Github
+
+_logger = logging.getLogger(__name__)
+
+
+class AbtractGithubModel(models.AbstractModel):
+    """
+    This abstract model is used to share all features related to github model.
+    Note that some fields and function have to be defined in the inherited
+    model. (github_type for instance)
+    """
+
+    _name = 'abstract.github.model'
+    _github_type = None
+    _github_login_field = None
+    _need_individual_call = False
+
+    github_id = fields.Char(
```

Comment:
Possible to use something other than `_id` for the naming? This alludes to a relation in normal Odoo naming & would throw me off in inherited models.

Maybe `github_id_external`?